### PR TITLE
BloomFilter implementation

### PR DESCRIPTION
**Pull request by chillaq**
# Python SDK

## Extra Notes

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     'flake8',
     'pytest==7.0.1',
     'pytest-mock>=3.5.1',
-    'coverage==6.2',
+    'coverage==6.1',
```
Comment (sanzmauro) [inline setup.py:11]: why do you downgrade this dependency?

File: splitio/engine/filters/base_filter.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import abc
+
+class BaseFilter(object, metaclass=abc.ABCMeta):
```
Comment (sanzmauro) [inline splitio/engine/filters/base_filter.py:3]: rename to `Filter` please

File: splitio/engine/filters/bloom_filter.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+from splitio.engine.filters.base_filter import BaseFilter
+from splitio import util
+from bloom_filter2 import BloomFilter
+
+class ImpressionsBloomFilter(BaseFilter):
```
Comment (sanzmauro) [inline splitio/engine/filters/bloom_filter.py:5]: rename to `BloomFilter` please

File: splitio/engine/filters/base_filter.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import abc
```
Comment (sanzmauro) [inline splitio/engine/filters/base_filter.py:1]: you are creating `__init__.py` file and is empty, is it correct? I think that you can define the interface in `__init__.py` and avoid to create this one.

**Review by sanzmauro — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (chillaq) [reply setup.py:11]: The tests were failing, I will change the version back to 6.2

File: splitio/engine/filters/base_filter.py (hunk 2 above)
Comment (chillaq) [reply splitio/engine/filters/base_filter.py:3]: This is how interfaces named in the SDK code

File: splitio/engine/filters/bloom_filter.py (hunk 3 above)
Comment (chillaq) [reply splitio/engine/filters/bloom_filter.py:5]: BlooomFilter is the 3rd party class name used, did not want to have collisions

File: splitio/engine/filters/base_filter.py (hunk 4 above)
Comment (chillaq) [reply splitio/engine/filters/base_filter.py:1]: will do

**Pull request merged**